### DISCUSSION TITLE
Progress bar not displaying

### DISCUSSION THREAD

**Discussion by roscrl** [unpoly/unpoly#380] [Q&A] [answered]
https://user-images.githubusercontent.com/13072760/168092217-526688e0-0561-4216-9a93-106ee568563d.mp4

```
<a href="/stock/${ticker.symbol()}" up-follow up-instant>
    <li class="hover:bg-black hover:text-white rounded-md px-2 mx-auto truncate">${ticker.name()}
        <span class="float-right">${ticker.symbol()}</span></li>
</a>
```

```up.network.config.progressBar = true``` enabled before booting

**Comment by roscrl** [comment]
Could this be a conflict with default styles of tailwind css?

**Comment by adam12** [comment, marked as answer]
Confirm the Unpoly version. And anything notable in the browser console log?

**Reply by roscrl** [reply]
2.5.3

![image](https://user-images.githubusercontent.com/13072760/168093467-0afac815-c3e9-4f48-b4c3-55c165f255bd.png)

**Reply by adam12** [reply]
The "already booted" console lines seem suspicious. How are you loading Unpoly?

**Reply by roscrl** [reply]
![image](https://user-images.githubusercontent.com/13072760/168094058-dc504ed9-2afd-4275-9296-cc6db65ecc8a.png)

Was going to make a separate issue on that but yes. I wasn't sure of the correct way to get it to properly boot. I am making use of importmaps with es-module-shims for Safari. Which I believe is causing the problem.

Removing ```<script async src="/webjars/es-module-shims/1.5.2/dist/es-module-shims.js"></script>``` & ```import "unpoly"``` & ```up.boot()``` works but breaks safari as the shim tries to load scripts twice.

**Reply by roscrl** [reply]
& using ```<script src="/webjars/unpoly/2.5.3/unpoly.js"></script>```  but does not solve progress bar issue

**Reply by adam12** [reply]
Are you loading the Unpoly CSS?

**Reply by roscrl** [reply]
Ooft apologies, copying the styles from unpoly.css to my main.css fixed the issue. What would the best way to import that to the assets folder?

**Reply by adam12** [reply]
I'd probably just use [`@import`](https://developer.mozilla.org/en-US/docs/Web/CSS/@import) or if I'm performance sensitive, another `<link>` in my header. Presuming that's what you mean?

**Reply by roscrl** [reply]
Yep of course, apologies. Thank you!
